# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/nsICookieManager.idl
# Commit: cc3c39aa8fb8
# Full Hash: cc3c39aa8fb8124cca265e84f71a29cbd218b704
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 2 - CookieStore API - IPDL, r=edgul,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215145
# ==============================================================================

diff -r 2fc9a3d0ad8b -r cc3c39aa8fb8 netwerk/cookie/nsICookieManager.idl
--- a/netwerk/cookie/nsICookieManager.idl	Wed Sep 11 16:26:54 2024 +0000
+++ b/netwerk/cookie/nsICookieManager.idl	Wed Sep 11 16:26:55 2024 +0000
@@ -229,6 +229,11 @@
   Array<nsICookie> getCookiesFromHost(in AUTF8String aHost,
                                       in jsval aOriginAttributes);
 
+  [notxpcom]
+  nsresult getCookiesFromHostNative(in AUTF8String aHost,
+                                    in OriginAttributesPtr aOriginAttributes,
+                                    out Array<nsICookie> aCookies);
+
   /**
    * Returns an array of all cookies whose origin attributes matches aPattern
    *
